nirc_ehr/resources/queries/dbo/q_prc_delete.sql

```diff
@@ -0,0 +1,74 @@
+SELECT substring(adt.PRIMARY_KEY_VALUES, length('ANIMAL_EVENT_ID = ')) AS objectid,
+       CAST(adt.CHANGE_DATETIME AS TIMESTAMP) AS modified,
+       adt.REFERENCE
+FROM AUDIT_TRAIL adt
+WHERE adt.PRIMARY_KEY_VALUES LIKE '%ANIMAL_EVENT_ID%' AND
+      adt.REFERENCE NOT LIKE '%Blood Sample Collection%' AND
+    (adt.REFERENCE LIKE '%Procedures%' OR
+     adt.REFERENCE LIKE '%Procedure%' OR
```

Comment:
This is redundant with the previous line. It may be best to leave both if they correspond to different event types to make the mapping as clear as possible, but again highlights the dangers of using LIKE clauses

Reply:
well there are multiple events including word "Procedures" and "Procedure", I thought it would be best to capture in a single LIKE as it makes sense for any event having Procedures to goto procedures.